pkg/machinery/config/types/v1alpha1/v1alpha1_examples.go

```diff
@@ -258,8 +258,9 @@ func machineTimeExample() *TimeConfig {
 
 func machineSysctlsExample() map[string]string {
 	return map[string]string{
-		"kernel.domainname":   "talos.dev",
-		"net.ipv4.ip_forward": "0",
+		"kernel.domainname":                   "talos.dev",
+		"net.ipv4.ip_forward":                 "0",
+		"net/ipv6/conf/eth0.100/disable_ipv6": "1",
```

Comment:
also here should be: `net.ipv6.conf.eth0/100.disable_ipv6`
This was my mistake

Reply:
I think it's correct according to the spec, but still my implementation was not correct:

```
Note that either "/" or "."  may be used as separators within
       sysctl variable names. If the first separator is a slash,
       remaining slashes and dots are left intact. If the first
       separator is a dot, dots and slashes are interchanged.
       "kernel.domainname=foo" and "kernel/domainname=foo" are
       equivalent and will cause "foo" to be written to
       /proc/sys/kernel/domainname. Either
       "net.ipv4.conf.enp3s0/200.forwarding" or
       "net/ipv4/conf/enp3s0.200/forwarding" may be used to refer to
       /proc/sys/net/ipv4/conf/enp3s0.200/forwarding
```

Reply:
we might provide either of as an example

Reply:
Ok I see, but this test should pass right? It doesn't anymore....

```
		{
			name: "Test Sysctl Path with vlan interface other notation",
			param: &kernel.Param{
				Key: kernel.Sysctl + ".net.ipv6.conf.eth0/103.disable_ipv6",
			},
			want: "/proc/sys/net/ipv6/conf/eth0.103/disable_ipv6",
		},
```

Reply:
yes, working on that!

Reply:
should be better now

Reply:
Yes, this is great!